tetrahedron.css

```diff
@@ -0,0 +1,8 @@
+#triangle polygon{
+    fill: transparent;
+    stroke: black;
+    stroke-width: 1%;
+}
+table, th, td {
+    border:1px solid black;
```

Comment:
For such a small amount of CSS code, is it necessary to have this CSS file, or could this logic be included in the .html file?